frontend/pages/model/[uuid].tsx

```diff
@@ -87,7 +86,6 @@ function Model() {
   const { versionAccess } = useGetVersionAccess(version?._id)
 
   const hasUploadType = useMemo(() => version !== undefined && !!version.metadata.buildOptions?.uploadType, [version])
-  const sendNotification = useNotification()
```

Comment:
Why has this been taken out - seems out of scope for this ticket?

Reply:
I don't remember doing this... I'll revert!